packages/manager/modules/carrier-sip/src/index.js

```diff
@@ -0,0 +1,30 @@
+import angular from 'angular';
+
+// Peer dependencies.
```

Comment:
I think comments should be used to say why you do something, not what you do, what do you think ? :)

Reply:
I have added them to keep organized as much as possible in all imports.

Nothing blocker but yes we can find maybe a way to harmonize all imports.

Reply:
IMO, comments to organize our code is still a nice things. It's not like it's comments that explain nothing like :

```
// Check if true
if (foo === true) {
}
```

Reply:
If we need the same comments on 100 files, it probably means we need another solution than comments, like a coding convention :)